src/sentry/static/sentry/app/views/organizationEventsV2/tagsTable.jsx

```diff
@@ -48,4 +55,4 @@ const TagValue = styled(TagKey)`
   text-align: right;
 `;
 
-export default TagsTable;
+export default withRouter(TagsTable);
```

Comment:
I would avoid using `withRouter` - it's caused with weird behavior for me before. I think what happens is that it will cause an additional render. When the routes change, we get an update due to the router and then `withRouter` will also update.

Reply:
Yeah, I was trying to avoid snaking the `location` down the render tree and also not using `window.location`.

If this re-renders when the URLs change I think that's ok here as this component is pretty lightweight.